### DISCUSSION TITLE
manager: pip_requirements: extend extra-index-url matcher

### DISCUSSION THREAD

**Discussion by Finkregh** [renovatebot/renovate#29726] [Suggest an Idea]
Re: https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/pip_requirements/common.ts#L38

This only matches `--extra-index-url http...`, but pip also works with `--extra-index-url=http...`, which lead to confusion.

https://pip.pypa.io/en/stable/reference/requirements-file-format/ does not mention `=` as separator, but it also works on the CLI and is (at least for me) expected behaviour for most CLI tools.

Should I create an issue or PR for this?

**Comment by rarkins** [comment]
PR is welcome

**Reply by Finkregh** [reply]
If you'd like to have a look: https://github.com/renovatebot/renovate/pull/29727

CI stuff is not green yet, so still WIP. Feel free to take over if I dont find the time to come back to this.

CLA is also signed, but also seems to take some time...
